package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name" : "okta-oauth-angular-samples",
+  "version" : "0.0.1",
+  "description" : "Sample OAuth2 AngularJS Examples",
+  "private" : true,
+  "main" : "server.js",
+  "scripts" : {
+    "start" : "node server.js | bunyan -o short"
+  },
+  "authors" : {
+    "Samples" : "Jordan Melberg",
+    "Server" : "Karl McGuinness"
+  },
+  "dependencies" : {
+    "body-parser": "^1.15.0",
```

Comment:
We should add @okta/okta-auth-js as a dependency here vs. checking it into source